pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -662,12 +666,14 @@ async def run_repomdrecord_download(name, location_href, downloader):
                     checksum_types[record.type] = record_checksum_type
                     record.checksum_type = record_checksum_type
 
-                    if self.mirror and (
-                        record.location_base
-                        or ".." in record.location_href
-                        or record.type == "prestodelta"
-                    ):
-                        raise ValueError(MIRROR_INCOMPATIBLE_REPO_ERR_MSG)
+                    if self.mirror:
+                        uses_base_url = record.location_base
+                        illegal_relative_path = (
+                            record.location_href.count("../") > self.namespace_depth
```

Comment:
I'm torn because I feel like the first thing someone looking at the code would do is go straight to `self.namespace_depth`, where they can see the full description.

Reply:
That's a fair observation, request withdrawn :)